apps/admin-portal/src/components/roles/create-role-wizard/role-user-assign.tsx

```diff
@@ -49,6 +53,18 @@ export const AddRoleUsers: FunctionComponent<AddRoleUserProps> = (props: AddRole
         getUsersList(limit, offset, filter, attribute, null)
             .then((response) => {
                 setUsersList(response.Resources);
+
+                if (assignedUsers && assignedUsers.length != 0) {
```

Comment:
Better to use a strict check here. i.e `!==`

Reply:
fixed with commit 0482c99